pkgs/development/python-modules/pytest_helpers_namespace/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, lib, buildPythonPackage, fetchPypi, isPyPy, python3, pytest }:
+
+with python3.pkgs;
```

Comment:
Its better to list all dependencies in the function that returns the derivation.